content/src/main/content/jcr_root/apps/acs-commons/widgets/source/js/multipanel.js

```diff
@@ -72,7 +72,7 @@ ACS.CQ.MultiFieldPanel = CQ.Ext.extend(CQ.Ext.Panel, {
                 return;
             }
 
-            pData[i.key] = i.getValue();
+            pData[i.key] = i.getValue().replace(/"/g, '&quot;');
```

Comment:
I've updated the code to do a thorough string check and only apply the
replace if it is a string.

-- Sent from my phone, appologies for thumb flubs and bad auto correct.
On Feb 12, 2015 4:38 PM, "david g." <notifications@github.com> wrote:

> In
> content/src/main/content/jcr_root/apps/acs-commons/widgets/source/js/multipanel.js
> <https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/416#discussion_r24622847>
> :
>
> > @@ -72,7 +72,7 @@ ACS.CQ.MultiFieldPanel = CQ.Ext.extend(CQ.Ext.Panel, {
> >                  return;
> >              }
> >
> > -            pData[i.key] = i.getValue();
> > +            pData[i.key] = i.getValue().replace(/"/g, '&quot;');
>
> Do you know if i.getValue() is always guaranteed to be a string (or an
> obj w a .replace method).
>
> Otherwise looks good! Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/416/files#r24622847>
> .
>